src/components/Modal/OfferDetailsModal.tsx

```diff
@@ -39,25 +46,71 @@ const OfferDetailsModal: React.FC<{ id: string }> = ({ id }) => {
                         <img src={closeIcon} className={styles.close_icon} />
                      </button>
                      <div className={styles.headline_area}>
-                        <img src={offerData.image} alt="image of offer" />
-                        {offerData.technologies.map((technology) => (
-                           <div key={technology}>{technology}</div>
-                        ))}
+                        <img
+                           className={styles.company_logo}
+                           src={offerData.image}
+                           alt="image of offer"
```

Comment:
**nitpick:** It would be good to add company title as `alt` instead of hardcoded value